src/wallet/signer.rs

```diff
@@ -665,8 +665,15 @@ pub struct SignOptions {
     /// Whether the signer should use the `sighash_type` set in the PSBT when signing, no matter
     /// what its value is
     ///
-    /// Defaults to `false` which will only allow signing using `SIGHASH_ALL`.
+    /// Defaults to `false` which will remove partial_sigs after input is signed
```

Comment:
It looks like you've accidentally modified `allow_all_sighashes` docs...

Reply:
🤦my bad